org.eclipse.scout.rt.shared/src/main/java/org/eclipse/scout/rt/shared/services/common/code/AbstractCodeTypeWithGeneric.java

```diff
@@ -443,11 +444,12 @@ protected void loadCodes() {
     // 1a add configured codes
     List<? extends CODE> createdList = interceptCreateCodes();
     if (createdList != null) {
-      for (CODE code : createdList) {
+      visit(createdList, (ICodeVisitor<CODE>) (code, treeLevel) -> {
```

Comment:
the following signature (without cast) would also compile and is a little bit more readable in my opinion:
`visit(createdList, (CODE code, int treeLevel) -> {`